src/components/Comment/Comment.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { typeComments } from '../../types';
+import './Comment.scss';
+
+export const Comment = ({ comment }) => (
+  <div className="comment">
+    <h4>{comment.name}</h4>
+    <p>{comment.body}</p>
+    <a href="https://www.gmail.com">{comment.email}</a>
+  </div>
+);
+
+Comment.propTypes = {
+  comment: typeComments.isRequired,
```

Comment:
```suggestion
  comment: PropTypes.Shape({
  postId: PropTypes.number.isRequired,
  id: PropTypes.number.isRequired,
  name: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
  body: PropTypes.string.isRequired,
}),
```

Reply:
Why should I do this? I'm sending {comment} as whole object